### PR TITLE
Use function get_yaml_item_value

### PR DESCRIPTION
**Pull request by jtydlack**
Enables using default even though the value was loaded as None.

### CODE REVIEW

**Review by tsebastiani — APPROVED** [review]
LGTM

File: run_kraken.py [hunk 1]
```diff
@@ -49,50 +50,77 @@ def main(cfg):
...
         kraken_config = cfg
-        chaos_scenarios = config["kraken"].get("chaos_scenarios", [])
-        publish_running_status = config["kraken"].get("publish_kraken_status", False)
+        chaos_scenarios = get_yaml_item_value(
+            config["kraken"], "chaos_scenarios", []
+        )
+        publish_running_status = get_yaml_item_value(
+            config["kraken"], "publish_kraken_status", False
+        )
         port = config["kraken"].get("port")
```
Comment (paigerube14) [inline run_kraken.py:66]: getting this error in the CI tests, lets go ahead and also set some sort of default value, maybe -1? @chaitanyaenr thoughts on default value? 

```
Traceback (most recent call last):
  File "/home/runner/work/krkn/krkn/run_kraken.py", line 517, in <module>
    main(options.cfg)
  File "/home/runner/work/krkn/krkn/run_kraken.py", line 165, in main
    if not 0 <= int(port) <= 65535:
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Comment (chaitanyaenr) [reply run_kraken.py:66]: Yes, we can set it to 8081 -  default in the krkn config.

File: run_kraken.py [hunk 2]
```diff
@@ -49,50 +50,77 @@ def main(cfg):
...
-        publish_running_status = config["kraken"].get("publish_kraken_status", False)
-        port = config["kraken"].get("port")
+        chaos_scenarios = get_yaml_item_value(
+            config["kraken"], "chaos_scenarios", []
+        )
+        publish_running_status = get_yaml_item_value(
+            config["kraken"], "publish_kraken_status", False
+        )
+        port = get_yaml_item_value(config["kraken"], "port", 8081)
         signal_address = config["kraken"].get("signal_address")
```
Comment (chaitanyaenr) [inline run_kraken.py:67]: let's set the default to 0.0.0.0

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**